### DISCUSSION TITLE
(v1.6 :اصدار) Kodi 19لـ Arctic Horizon 1 بيلد عربي

### DISCUSSION THREAD

**Discussion by psp2111-ADSLGATE** [psp2111-ADSLGATE/psp2111-ADSLGATE.github.io#4] [General]
# <p align="center"> بسم الله الرحمن الرحيم </p>

# التغييرات:
### اصدار 1.6:

- إضافة 11 خط للترجمة ممكن الاختيار منها في "الاعدادات" ثم "مشغل الوسائط" ثم " "اللغة" ثم "خط الترجمة"
- تحسين في أداء TMDb Helper و Seren أثناء عملية البحث
- اصلاح مشكلة عدم توفر العرض الدعائي او Trailer لكل الافلام والمسلسلات
- اصلاح مشكلة قلة او عدم توفر المحتوى في قسم التصنيفات مثل "رعب" و "غربي"
- اصلاح مشكلة وضوح العرض دعائي على 720p ورفعها الى 4K او 1080p (حسب المتوفر من المصدر)
- تفعيل التحميل التلقائي للترجمة (غير دقيق ممكن يصيب او يخطي)

### اصدار 1.5:
...

### اصدار 1.4:

- تحسينات في الاداء وسرعة تحميل الويدجيت.
- تم تقليل عدد الويدجيت في الواجهة من 30 الى 20 لكل عنصر.
- الان أغلب معلومات الافلام والمسلسلات من عناوين ووصف باللغة العربية وتعمل بشكل صحيح مع محرك بحث Seren .
- تم تغيير مصدر الويدجيت من Seren الى TMDB Helper.

## متطلبات التشغيل:

- جهاز بمعالج قوي مثل الشيلد أو أقوى
- رام 3 جيجا أو أكثر
- حساب [Real Debrid](http://real-debrid.com/?id=2815547) مدفوع ومفعل
- حساب في موقع [Trakt](https://trakt.tv/) مجاني
- مفتاح [OMDb API](https://www.omdbapi.com/apikey.aspx?__EVENTTARGET=freeAcct&__EVENTARGUMENT=&__LASTFOCUS=&__VIEWSTATE=%2FwEPDwUKLTIwNDY4MTIzNQ9kFgYCAQ9kFgICBw8WAh4HVmlzaWJsZWhkAgIPFgIfAGhkAgMPFgIfAGhkGAEFHl9fQ29udHJvbHNSZXF1aXJlUG9zdEJhY2tLZXlfXxYDBQtwYXRyZW9uQWNjdAUIZnJlZUFjY3QFCGZyZWVBY2N0oCxKYG7xaZwy2ktIrVmWGdWzxj%2FDhHQaAqqFYTiRTDE%3D&__VIEWSTATEGENERATOR=5E550F58&__EVENTVALIDATION=%2FwEdAAU%2BO86JjTqdg0yhuGR2tBukmSzhXfnlWWVdWIamVouVTzfZJuQDpLVS6HZFWq5fYpioiDjxFjSdCQfbG0SWduXFd8BcWGH1ot0k0SO7CfuulHLL4j%2B3qCcW3ReXhfb4KKsSs3zlQ%2B48KY6Qzm7wzZbR&at=freeAcct&Email=) لتفعيل التقييمات ( اختياري )

## التحميل و التثبيت:
- [شرح طريقة التحميل و التثبيت](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/discussions/2)
 
## صور البليد:
![1tlXkxG](https://user-images.githubusercontent.com/80945174/188753879-edce0e7e-0dc8-4ba8-a7da-04b8765b4ed3.png)
![qAQoSpN](https://user-images.githubusercontent.com/80945174/188753889-f178c367-58b3-4239-be85-72b4414aff54.png)
![latYadg](https://user-images.githubusercontent.com/80945174/188753895-4df234bd-f691-476b-914d-eb405792286f.png)
![mm3jVRJ](https://user-images.githubusercontent.com/80945174/188753897-2dbfd132-7691-4d60-bc2a-71e1c969eb85.png)
![EiipFPg](https://user-images.githubusercontent.com/80945174/188753909-6ea2de24-5d39-44db-8481-73c145986755.png)
![u3RuhJi](https://user-images.githubusercontent.com/80945174/188753915-f06291ed-5b84-4f81-b634-8b6b70c0ac96.png)
![yuD3JX3](https://user-images.githubusercontent.com/80945174/188753920-f51239c8-7980-469d-b3ce-dadef8b10a85.png)
![K8uIHIh](https://user-images.githubusercontent.com/80945174/188753927-ae2c9e22-c002-438b-888a-fd033e8e2e94.png)
![nMEKNgr](https://user-images.githubusercontent.com/80945174/188753931-67ed9cee-d018-4484-88ed-db7e5d5a145e.png)
![K4FxwpD](https://user-images.githubusercontent.com/80945174/188753936-27ede368-48c5-4368-bb10-8298258b5e3c.png)
![pVZYsSR](https://user-images.githubusercontent.com/80945174/188753941-4f5c0f59-fefe-4b11-90b7-c7cc03a15e33.png)

**Comment by ghost** [comment]
اضافة جميلة ومجهود تشكر عليه لكن بذكر تجربتي وبعض الملاحظات بكل حيادية برنامج كودي معروف انه ثقيل ومع هذي الاضافة صار كارثي كراشات عشوائية وفترة انتظار لا نهائية لو ترجمت الشكل الاصلي للتطبيق seren كان افضل لانه خفيف وسلس
ومع خلفيات الطبيعة خلت التطبيق كانه شي رخيص ولو تكتفي بلونين راح تخلي للاضافة رونق وفخامة مثل اللي تسويه نتفليكس خلفية سوداء وخط احمر

**Reply by psp2111-ADSLGATE** [reply]
هذا Build وليس اضافة وهو عبارة عن مجموعة اضافات وتغييرات في تطبيق كودي قمت بها

بالنسبة لو كنت تفضل الشكل الاصلي لكودي معرب اضف المصدر : https://psp2111-adslgate.github.io/ 
الى كودي وثبت الاصدار المعدل من Estuary (سكن كودي الاصلي)
بعد كذا غير الخط من اعدادات interface الى Samim
ارجع الى المستودع psp2111 ثم ثبت اضافة التعريب وبكذا راح يتعرب معك كودي

اذا كنت تفضل Build مشابه لشكل netflix حمل نسخة Cosmic هي الاقرب لثيم Netflix
على العموم انصحك تجرب Arctic Horizon 2 لان هي التي استخدمها بشكل يومي

**Comment by nx107** [comment]
السلام عليكم ورحمة الله 
الاضافة جميلة ومرتبة وكل شي فيها ممتاز اشكرك صراحة 
بس عندي مشكلة في الصورة يكون فيها تقطيع ليس سببه الانترنت والجهاز الي عندي بي سي حديث والشاشة من LG GX
الكيبل من ugreen 2.1 بس ما ادري وش سبب التقطيع حاولت القى حل وماتوصلت لحل ممكن تساعدني

**Reply by psp2111-ADSLGATE** [reply]
وعليكم السلام 
يفضل تنضم على قروب التلجرام https://t.me/+sRwFCWMT8y4zMTg0
وتقدر بعدها تتواصل معي على الخاص أو تسأل في القروب
